newdle/client/src/components/NewdleHeader.js

```diff
@@ -0,0 +1,34 @@
+import React, {useEffect} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {Container, Label} from 'semantic-ui-react';
+import {getNewdleTitle, getUserInfo, getNewdleFinalDt} from '../selectors';
+import {fetchNewdle} from '../actions';
+import styles from './NewdleHeader.module.scss';
+
+export default function NewdleHeader({match}) {
+  const code = match.params.code;
+  const title = useSelector(getNewdleTitle);
+  const user = useSelector(getUserInfo);
```

Comment:
that's not correct, it returns your own user info, not the newdle creator. we don't store the name of the newdle creator so far (just an identifier), so I'd either fully omit it or put a hardcoded dummy name below (and add a task to store the creator's name when creating a newdle)

Reply:
I used this because I thought that this summary page will only be seen by the currently logged in user (you can only see the summary of your personal newdles)

Reply:
ok, good point!

but then the whole message is a bit useless IMHO

Reply:
It might be useless in the end. I was following the mockups and imo without this the title looks a bit empty.